src/content/docs/zh-cn/distribute/rpm.mdx

```diff
@@ -0,0 +1,339 @@
+```
+
+3. 向脚本添加一些内容。
+
+```bash title="preinstall.sh"
+echo "-------------"
+echo "This is pre"
+echo "Install Value: $1"
+echo "Upgrade Value: $1"
+echo "Uninstall Value: $1"
+echo "-------------"
+```
+
+```bash title="postinstall.sh"
+echo "-------------"
+echo "This is post"
+echo "Install Value: $1"
+echo "Upgrade Value: $1"
+echo "Uninstall Value: $1"
+echo "-------------"
+```
+
+```bash title="preremove.sh"
+echo "-------------"
+echo "This is preun"
+echo "Install Value: $1"
+echo "Upgrade Value: $1"
+echo "Uninstall Value: $1"
+echo "-------------"
+```
+
+```bash title="postremove.sh"
+echo "-------------"
+echo "This is postun"
+echo "Install Value: $1"
+echo "Upgrade Value: $1"
+echo "Uninstall Value: $1"
+echo "-------------"
+```
+
+4. 将脚本添加到 `tauri.conf.json` 文件中
+
+```json title="tauri.conf.json"
+{
+  "bundle": {
+    "linux": {
+      "rpm": {
+        "epoch": 0,
+        "files": {},
+        "release": "1",
+        // add the script here
+        "preInstallScript": "/path/to/your/project/src-tauri/scripts/prescript.sh",
+        "postInstallScript": "/path/to/your/project/src-tauri/scripts/postscript.sh",
+        "preRemoveScript": "/path/to/your/project/src-tauri/scripts/prescript.sh",
+        "postRemoveScript": "/path/to/your/project/src-tauri/scripts/postscript.sh"
+      }
+    }
+  }
+}
+```
+
+### 设置 Conflict、 Provides、 Depends、 Files、 Obsoletes、 DesktopTemplate 和 Epoch
+
+- **conflict**: 防止安装与另一个包冲突的包。
+  例如，如果您更新了应用程序所依赖的 RPM 包，但新版本与您的应用程序不兼容。
+
+- **provides**: 列出应用程序提供的 RPM 依赖项。
+
+- **depends**: 列出应用程序运行所需的 RPM 依赖项。
+
+- **files**: 指定包中包含的文件。
+
+- **obsoletes**: 列出您的应用程序废弃的 RPM 依赖项。
+
+:::note
+如果安装了这个软件包，那么被列为“过时”的软件包将会在存在的情况下自动删除。
+:::
+
+- **desktopTemplate**: 向包中添加自定义桌面文件。
+
+- **epoch**: 基于版本号定义加权依赖关系。
+
+:::caution
+除非必要，否则不建议使用 epoch，因为它会改变包管理器比较包版本的方式。
+有关epoch的更多信息，请查看：[RPM 打包指南](https://rpm-packaging-guide.github.io/#epoch-scriptlets-and-triggers).
+:::
+
+要使用这些选项，请将以下内容添加到你的 `tauri.conf.json` 文件中。
+
+```json title="tauri.conf.json"
+{
+  "bundle": {
+    "linux": {
+      "rpm": {
+        "postRemoveScript": "/path/to/your/project/src-tauri/scripts/postscript.sh",
+        "conflicts": ["oldLib.rpm"],
+        "depends": ["newLib.rpm"],
+        "obsoletes": ["veryoldLib.rpm"],
+        "provides": ["coolLib.rpm"],
+        "desktopTemplate": "/path/to/your/project/src-tauri/desktop-template.desktop"
+      }
+    }
+  }
+}
+```
+
+### 给包添加一个许可证
+
+要给包添加许可证，请将以下内容添加到 `src-tauri/cargo.toml` 或者 `src-tauri/tauri.conf.json` 文件中：
+
+```toml title="src-tauri/cargo.toml"
+[package]
+name = "tauri-app"
+version = "0.0.0"
+description = "A Tauri App"
+authors = ["you"]
+edition = "2021"
+license = "MIT" # add the license here
+# ...  rest of the file
+```
+
+对于 `src-tauri/tauri.conf.json`
+
+```json title="src-tauri/tauri.conf.json"
+{
+  "bundle": {
+    "licenseFile": "../LICENSE", // put the path to the license file here
+    "license": "MIT" // add the license here
```

Comment:
```suggestion
    "licenseFile": "../LICENSE", // 将许可证文件路径放在此
    "license": "MIT" // 将许可证添加在此
```
Again, since the comments here are directly tied to the instruction so it's probably better to translate it out.